tools/build_variables.bzl

```diff
@@ -596,6 +596,7 @@ libtorch_python_core_sources = [
     "torch/csrc/utils.cpp",
     "torch/csrc/utils/cuda_lazy_init.cpp",
     "torch/csrc/utils/invalid_arguments.cpp",
+    "torch/csrc/utils/crash_handler.cpp",
```

Comment:
Do you know what this list is doing and why we are not importing it from CMAKE?

Reply:
We do read from it

https://github.com/pytorch/pytorch/blob/c8cf9114bf79405b2888cd2542b55f78bf3773fc/cmake/Codegen.cmake#L249-L252

https://github.com/pytorch/pytorch/blob/c8cf9114bf79405b2888cd2542b55f78bf3773fc/torch/CMakeLists.txt#L45

Reply:
Hmm, that why do you need to add it explicitly on line 582 in caffe2/CMakeLists.txt ?